src/main/java/co/cask/plugin/format/MultiTableConf.java

```diff
@@ -48,15 +51,23 @@
 
   @Macro
   @Nullable
-  @Description("A pattern that defines which tables should be read from. " +
-    "Any table whose name matches the pattern will read. If not specified, all tables will be read.")
-  private String tableNamePattern;
+  @Description("A schema name to list all the tables from. By default all the schemas will be used in the listing call.")
+  private String schemaName;
```

Comment:
from jdbc docs, this looks like a pattern, should rename to schemaPattern and update docs.

Reply:
Updated.